src/autotype/AutoType.cpp

```diff
@@ -102,6 +103,19 @@ void AutoType::loadPlugin(const QString& pluginPath)
     }
 }
 
+void AutoType::unloadPlugin()
+{
+    if (m_executor) {
+        delete m_executor;
+        m_executor = nullptr;
```

Comment:
Change m_executor to a QPointer and remove nullptr assignments.

Reply:
AutoTypeExecutor doesn't inherit from QObject so you can't use QPointer here

Reply:
Any strong objections against changing that?

Reply:
Seems that this triggers some error with the linker

I would prefer changing this in a follow up PR

Reply:
Shouldn't cause linker errors.